<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriftCalculationPage</class>
 <widget class="QWidget" name="DriftCalculationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>起始点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>终止点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>漂移量</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
       <item>
        <widget class="QLineEdit" name="lineEdit_startPoint1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_endPoint1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_drift1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
       <item>
        <widget class="QLineEdit" name="lineEdit_startPoint2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_endPoint2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_drift2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
       <item>
        <widget class="QLineEdit" name="lineEdit_startPoint3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_endPoint3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_drift3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1,1">
       <item>
        <widget class="QLineEdit" name="lineEdit_startPoint4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_endPoint4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_drift4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_StartCalcul">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Cancel">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
